layouts/shortcodes/multifilter-search.html

```diff
@@ -0,0 +1,285 @@
+                                    {{ partial "icon" (dict "name" "close-light_x" "title" "delete" "size" "14px") }}
+                                </div>
+                            </div>
+                        </template>
+
+                    </div>
+                    {{/*  Delete all selections for a filter type  */}}
+                    <div class="overflow-gradient h-100 position-absolute top-0 end-0 rounded-end">
+                        <template x-if="filterTypes['{{$filter_type}}'].length">
+                            <span class="cancel-circle-wrapper" @click.stop="resetFilteredSet; deleteAllSelections('{{ $filter_type }}')">
+                                {{ partial "icon" (dict "name" "x-circle-solid" "title" "delete all" ) }}
+                            </span>
+                        </template>
+                        <span class="chevron-wrapper d-flex align-items-center justify-content-center position-relative h-100">
+                            <div class="chevron chevron-down"></div>
+                        </span>
+                    </div>
+                </button>
+                {{/*  FILTER FORM  */}}
+                <form 
+                data-id="{{$filter_type}}"
+                class="d-flex flex-column bg-white rounded position-absolute w-100"
+                x-cloak x-show.important="openedFilterForm === '{{$filter_type}}'"
+                @click.outside="openedFilterForm = null"
+                >
+                    <div class="filter-selection-container">
+                        <span>All</span>
+                    </div>
+
+                    {{ range (index ($navigation.Get "dropdown") $h.name) | sort  }}
+                      {{ $id := (printf "%s-%s" . $filter_type) | anchorize }}
+                      <div class="filter-selection-container">
+                          <input 
+                          id='{{ $id }}' 
+                          type="checkbox" 
+                          value='{{ . }}' 
+                          x-model="filterTypes['{{$filter_type}}']"
+                          @change="resetFilteredSet"
+                          @change.debounce="pushState('{{ $filter_type }}', '{{ . }}')"
+                          >  
+                          
+                          <label for="{{ $id }}" class="d-flex align-items-center">
+                              {{.}}
+                          </label>
+                      </div>
+                    {{ end }}
+                </form>
+            </div>                      
+          
+        </div>
+        {{ end }}
+    </div>
+    {{/*  SEARCH FORM */}}
+    <form class="search__multifilter" @submit.prevent="">
+        <input type="text" placeholder="Search here" class="w-100 px-1" x-model="searchValue" @input="resetFilteredSet" @input.debounce="pushState()">
```

Comment:
```suggestion
        <input type="search" placeholder="Search here" class="w-100 px-1" x-model="searchValue" @input="resetFilteredSet" @input.debounce="pushState()">
```
this might cause an unwanted search icon to appear, but just pushing for semantic HTML